src/helpers/setupDocViewer.js

```diff
@@ -17,6 +17,7 @@ const addToolsToDocViewer = docViewer => {
   toolModeMap.AnnotationCreateFreeHand2 = new window.Tools.FreeHandCreateTool(docViewer, 'AnnotationCreateFreeHand2');
   toolModeMap.AnnotationCreateFreeHand3 = new window.Tools.FreeHandCreateTool(docViewer, 'AnnotationCreateFreeHand3');
   toolModeMap.AnnotationCreateFreeHand4 = new window.Tools.FreeHandCreateTool(docViewer, 'AnnotationCreateFreeHand4');
+  toolModeMap.AnnotationCreateEraser = new window.Tools.EraserTool(docViewer, 'AnnotationCreateEraser');
```

Comment:
Prefer doing this in WebViewer instead. Please add this tool in `DocumentViewer.js` ~line 246, `this.toolModeMap`.

The freehand tools and highlight tools above only exist for the purpose of the current UI design

Reply:
Ohh ok, I'll do that!